source/bdDetect.py

```diff
@@ -153,13 +152,18 @@ def getDriversForConnectedUsbDevices(
 		for port in deviceInfoFetcher.hidDevices
 		if port["provider"] == "usb"
 	))
+
+	fallbackDriversAndMatches = []

```

Comment:
Can you fill out this type hint please

```suggestion
	fallbackDriversAndMatches: list[set[str, DeviceMatch]] = []
```